src/styles/all-styles/normalize.scss

```diff
@@ -0,0 +1,94 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #{$white-color};
+}
+
+html,
+body,
+div,
+span,
+applet,
+object,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
```

Comment:
Write for those tags that are used in the project, you don't need to enter all the existing ones here